torchvision/ops/misc.py

```diff
@@ -46,7 +48,12 @@ class FrozenBatchNorm2d(torch.nn.Module):
     are fixed
     """
 
-    def __init__(self, num_features, eps=0., n=None):
+    def __init__(
+        self,
+        num_features: Tuple[int, ...],
```

Comment:
nit: I would have expected `num_features` to be an `int`, not a tuple with a varying number of elements.